Automattic/wp-calypso
# FSE: expose additional themes in signup

#### Changes proposed in this Pull Request

Adds Morden, Stow, and Hever to the theme selection step
in the signup flow.

<img width="796" alt="Screenshot 2019-12-04 at 16 38 15" src="https://user-images.githubusercontent.com/1182160/70161782-8df83600-16b4-11ea-99f7-3bcf185e529d.png">


#### Testing instructions

1. Navigate to http://calypso.localhost:3000/start/test-fse/fse-themes
2. Verify that you can see all of the newly supported FSE themes: Alves, Maywood, Morden, Stow, Hever, Exford, Shawburn.
